elections/2020/governance-committee-election.md

```diff
@@ -0,0 +1,180 @@
+committee members may not run in the election, though they may vote. We strive
+for transparency in the election process and hold the interests of the
+community as our first priority. In particular, we will ensure that all
+documents and assets related to the 2020 election process are public, and we
+will attempt to record and distribute notes for any meetings held as part of
+this process.
+
+# Nominations
+
+As per
+[the charter document](https://github.com/open-telemetry/community/blob/master/governance-charter.md#establishment-of-a-governance-committee),
+anybody is eligible to run for the Governance Committee. During the “call for
+nomination” period, people can be nominated or nominate themselves by
+submitting a Pull Request adding said candidate to the
+[governance-committee-candidates.md](https://github.com/open-telemetry/community/blob/master/elections/2020/governance-committee-candidates.md)
+file in the OpenTelemetry community repository. The template in that file includes
+the following columns:
+* Full name
+* GitHub alias
+* Company affiliation
+* Short bio and/or reasoning to join the Governance Committee (no more than a
+short paragraph)
+* _Optional_: photo/picture of a nominee
+
+The Pull Request will not be merged until the candidate has confirmed their
+desire to be nominated (if not self-nominating) and ratification has occurred
+via PR comments.
+
+The email address of the nominee should also be sent to
+[cncf-opentelemetry-governance@lists.cncf.io](mailto:cncf-opentelemetry-governance@lists.cncf.io) –
+it will be kept private and used only for candidate communications as the
+election process proceeds.
+
+The Governance Committee or appointed people will contact every nominee
+directly to ensure the commitment and desire to be nominated.
+
+# Voter pre-registration
+
+All active GitHub contributors will receive a voting link to the email
+associated with their commits. If your code contributions do not meet
+eligibility requirements, but you are eligible for voting with the non-code
+contributions, you need to pre-register so a voting link can be sent.
+
+The form will ask for email and a proof of eligibility for the vote. Criteria
+for voting eligibility are listed in the Members of Standing section of the
+[charter document](https://github.com/open-telemetry/community/blob/master/governance-charter.md#members-of-standing).
+
+One of two options will be available in the form to prove eligibility:
+
+* GitHub handle
+* List either…
+  * Your contributions to OpenTelemetry that make you eligible to vote, or
+  * Your non-code contributions per the [Members of Standing section of the
+charter](https://github.com/open-telemetry/community/blob/master/governance-charter.md#members-of-standing):
+particularly, your relationship to a well-known project that’s taken a hard
+dependency on OpenTelemetry, and/or your involvement with a well-known
+organization’s adoption of OpenTelemetry as an end-user.
```

Comment:
styling nit: I prefer spacing in lists paragraphs. It renders OK on GitHub, but may not render well on other platforms =)

```suggestion
    charter](https://github.com/open-telemetry/community/blob/master/governance-charter.md#members-of-standing):
    particularly, your relationship to a well-known project that’s taken a hard
    dependency on OpenTelemetry, and/or your involvement with a well-known
    organization’s adoption of OpenTelemetry as an end-user.
```